### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by petermattson**
Moving extensive project info into Github.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,119 @@
...
+* Metadata: description of the dataset, including responsible ML aspects
+* Resources: one or more files or other sources containing the raw data
+* Structure: how the raw data is combined and arranged into data structures for use
+* ML semantics: how the data is most often used in an ML context
+
+## Simple Example
+
+Here is an extremely simple example of the croissant format, with comments showing the four layers:
+
+```
```
Comment (pierrot0) [inline README.md:25]: You can replace code block with: https://github.com/mlcommons/croissant/blob/e9bbcbd903372339a951c328e8172b7b82582684/datasets/recipes/minimal_recommended.json#L1-L50

Which will render the code properly., similarly to https://github.com/mlcommons/croissant/blob/main/docs/howto/specify-splits.md.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,119 @@
...
+## Integrations
+Forthcoming
+
+## Licensing
+Croissant project code and examples are licensed under Apache 2.
+
+## Governance
+Croissant is being developed by the community as a Task Force of the [MLCommons Association](mlcommons.org) Datasets Working Group.
+The Task Force is open to anyone (as is the parent working group). 
+The Task Force is co-chaired by [Omar Benjello](mailto:benjello@google.com) and TBD.
```
Comment (marcenacp) [inline README.md:114]: Omar Benjelloun

File: README.md [hunk 3]
```diff
@@ -1,13 +1,119 @@
...
+
+## Contributors
+Albert Villanova (Hugging Face), Andrew Zaldivar (Google), Baishan Guo (Meta), Carole Jean-Wu (Meta), Ce Zhang (ETH Zurich), Costanza Conforti (Google), D. Sculley (Kaggle), Dan Brickley (Schema.Org), Eduardo Arino de la Rubia (Meta), Edward Lockhart (Deepmind), Elena Simperl (King's College London), Geoff Thomas (Kaggle), Joaquin Vanschoren (TU/Eindhoven, OpenML), Jos van der Velde (TU/Eindhoven, OpenML), Julien Chaumond (Hugging Face), Kurt Bollacker (MLCommons), Lora Aroyo (Google), Meg Risdal (Kaggle), Natasha Noy (Google), Newsha Ardalani (Meta), Omar Benjelloun (Google), Peter Mattson (MLCommons), Pierre Marcenac (Google), Pierre Ruyssen(Google), Pieter Gijsbers (TU/Eindhoven, OpenML), Prabhant Singh (TU/Eindhoven, OpenML), Quentin Lhoest (Hugging Face), Taniya Das (TU/Eindhoven, OpenML)
```
Comment (marcenacp) [inline README.md:117]: Missing space between `Pierre Ruyssen` and `Google`

**Review by pierrot0 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (marcenacp) [inline README.md:25]: Add ```json for a better formatting

**Pull request merged**